recipes/mmlong2-lite/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mmlong2-lite" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "66cc1063df513697cafb524698e4125a5cffa2e63f80c209d10ee3e041049601" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serka-M/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('mmlong2-lite', max_pin="x.x") }}
+
+requirements:
+  run:
+    - snakemake 8.2.3
```

Comment:
Does it need exactly these versions ?
I guess it will still work with newer versions too ?! Especially for `tar`
Can we relax the pins ?

Reply:
With Snakemake, I'd rather stick to a specific version since it gets changed frequently, causing incompatibilities.
The Singularity version is needed and the rest can work with newest versions, although my employer is very adamant about version-controlling everything.